src/servers/grpc_server.cc

```diff
@@ -679,26 +679,26 @@ InferResponseAlloc(
         *actual_memory_type_id = pr->second.device_id_;
         use_shm = true;
 
-        LOG_VERBOSE(1) << "GRPC shared-memory: " << tensor_name << ", size "
-                       << byte_size << ", addr " << *buffer;
+        LOG_VERBOSE(1) << "GRPC: using shared-memory for " << tensor_name
```

Comment:
How about wrapping tensor name in single quotes and using a ":" after size and addr ?